From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Wed, 5 Sep 2018 09:18:41 +0200
Subject: [PATCH] Linux 4.4.154
References: bnc#1012382
Patch-mainline: 4.4.154
Git-commit: 4d9339a1f0e6e288903d34020f2378d250ea3cac

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 208a813be615..b184286cf7e6 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,6 @@
 VERSION = 4
 PATCHLEVEL = 4
-SUBLEVEL = 153
+SUBLEVEL = 154
 EXTRAVERSION =
 NAME = Blurry Fish Butt
 
-- 
2.18.0

