From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Fri, 8 Feb 2019 11:25:33 +0100
Subject: [PATCH] Linux 4.4.174
References: bnc#1012382
Patch-mainline: 4.4.174
Git-commit: dc5e8c99975bb1a1561de884a83b3c19e4ac7ada

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index db7665e32da8..1fa281069379 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,6 @@
 VERSION = 4
 PATCHLEVEL = 4
-SUBLEVEL = 173
+SUBLEVEL = 174
 EXTRAVERSION =
 NAME = Blurry Fish Butt
 
-- 
2.20.1

