From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Wed, 10 Oct 2018 08:52:13 +0200
Subject: [PATCH] Linux 4.4.160
References: bnc#1012382
Patch-mainline: 4.4.160
Git-commit: 11abaca7d5b923f1a14c578746d4232bc4ca7d72

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 06d5c6a6a0f6..607394a56036 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,6 @@
 VERSION = 4
 PATCHLEVEL = 4
-SUBLEVEL = 159
+SUBLEVEL = 160
 EXTRAVERSION =
 NAME = Blurry Fish Butt
 
-- 
2.19.1

